recipes/pycifrw/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "PyCifRW" %}
+{% set version = "4.3" %}
+{% set sha256 = "3b19e80278b40ebe00ad49a7dd8813007fbdc7a9a43fd3377e404ac0bcbcd3c3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - toolchain
+  run:
+    - python
+
+test:
+  imports:
+    - CifFile.CifFile
+    - CifFile.StarFile
+    - CifFile.StarScan
+
+about:
+  home: https://bitbucket.org/jamesrhester/pycifrw/overview
+  license: PSF-2-like
+  license_family: other
+  license_file: LICENSE
+  summary: 'CIF file support for Python'
+
+  description: |
+    PyCIFRW provides support for reading and writing CIF (Crystallographic
+    Information Format) files using Python. It was developed at the Australian
+    National Beamline Facility (ANBF), run by the Australian Synchrotron
+    Research Program (ASRP), as part of a larger project to provide CIF
+    input/output capabilities for data collection. It is now (Mar 2017)
+    maintained and developed within the Australian Nuclear Science and
+    Technology Organisation (ANSTO).
+  doc_url: https://bitbucket.org/jamesrhester/pycifrw/overview
+  dev_url: https://bitbucket.org/jamesrhester/pycifrw/overview
+
+extra:
+  recipe-maintainers:
+    - CJ-Wright
+    - pavoljuhas
```

Comment:
Yes, I am fine with that.  I'd like to notify package author to see if he has any comments on having this on conda forge.